SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialPlayerSpawner.h

```diff
@@ -12,6 +12,8 @@
 #include <WorkerSDK/improbable/c_schema.h>
 #include <WorkerSDK/improbable/c_worker.h>
 
+#include "EntityQueryHandler.h"
```

Comment:
nit: include should be up with the other spatial includes

Reply:
Certainly - it's not shown in this comment as it's attached to a previous version but I've fixed this.